scripts/prep-for-goreleaser.sh

```diff
@@ -1,22 +1,53 @@
 #!/bin/bash
 
+# When run by goreleaser, FILTER_OS and DEST_DIR are blank, so files are installed in:
+#
+# * ./bin/darwin
+# * ./bin/linux
+# * ./bin/windows
+#
+# Allowing us to customize the archives for each.
+#
+# When run by GitHub Actions in tests, we set the first arg, FILTER_OS, so that
```

Comment:
Clever.. Wondering if way too clever, perhaps could've forked the script instead. But I'm ok with this I appreciate where the need for this munging is coming from.